mycli/myclirc

```diff
@@ -86,6 +86,11 @@ Token.Menu.Completions.MultiColumnMeta = 'bg:#aaffff #000000'
 Token.Menu.Completions.ProgressButton = 'bg:#003333'
 Token.Menu.Completions.ProgressBar = 'bg:#00aaaa'
 
+# Query results
+Token.Output.Header = 'bold'
+Token.Output.OddRow = ''
+Token.Output.EvenRow = ''
```

Comment:
Background colors also don't work so well because there's padding. See an example:
<img width="351" alt="screenshot 2017-06-27 21 02 30" src="https://user-images.githubusercontent.com/637013/27620095-0ce32930-5b7c-11e7-9edd-bdb7b79be033.png">

This is with:

```
Token.Output.Header = 'bold'
Token.Output.OddRow = '#ansiteal'
Token.Output.EvenRow = 'bg:#cccccc #ansiblack'
```

It's a shame really. I wonder if we can fix that.